.github/workflows/linting.yml

```diff
@@ -23,9 +23,14 @@ jobs:
 
       - name: Install dependencies
         run: pip install isort black==22.3.0
```

Comment:
perhaps leaving the black free of version could also benefit flamby (unless there is a reason for having this exact version)?

Reply:
I agree it's kind of weird but let's leave that for future works because it is part of a larger topic of: should we pin versions in pre-commit hooks as well or not yada yada yada.